frontend/src/pages/HomePage.jsx

```diff
@@ -44,17 +45,21 @@ class HomePage extends Component {
     menuOpen: false,
     registerMessage: false,
     registerSuccess: false,
-    loginSuccess: false
+    loginSuccess: false,
+    loginMessage: false,
+    loginError: false
   }
 
   componentDidMount() {
     this.props.setRegistering(false)
+    this.props.setLogging(false)
     this.props.setRegister_dialog(false)
     this.props.setLogin_dialog(false)
   }
 
   componentWillUnmount() {
-    this.props.setStatus(false);
+    // this.props.setStatus(false);
+    // this.props.setToken("")
   }
```

Comment:
Picky: Remove the whole lifecycle method if not used(?

Reply:
Yeah, I’ll remove it later